typedoc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "entryPoints": [
+        "src/extension.ts",
+        "src/core/commandRunner/CommandRunner.ts",
+        "src/core/inferFullTargets.ts",
+        "src/core/updateSelections/RangeUpdater.ts",
+        "src/core/updateSelections/updateRangeInfos.ts",
+        "src/core/updateSelections/updateSelections.ts",
+        "src/processTargets/index.ts",
+        "src/processTargets/modifiers/processModifier.ts",
+        "src/processTargets/processMark.ts",
+        "src/typings/Types.ts",
+        "src/typings/treeSitter.ts",
+        "src/util/canonicalizeAndValidateCommand.ts",
+        "src/util/getPrimitiveTargets.ts"
+    ],
+    "readme": "none", 
+    "excludePrivate": true,
+    "out": "docs/contributing/api"
```

Comment:
Given we are just doing this transiently, maybe let's put it in a dir that makes it look more auto-generated, and then add it to `.gitignore`?  Maybe `build/docs/api`?  I was thinking `docs/contributing/api` only if we were actually planning to commit the docs to source

Reply:
Although on second thought, maybe we want to just publish the whole `docs` dir?  In that case I think let's stick with this directory, but add it to `.gitignore`